docs/source/markdown/podman-generate-systemd.1.md

```diff
@@ -96,9 +136,82 @@ PIDFile=/run/user/1000/overlay-containers/ccfd5c71a088768774ca7bd05888d55cc28769
 WantedBy=multi-user.target default.target
 ```
 
+### Installation of generated systemd unit files.
+
+Podman generated unit files include an `[Install]` section, which carries installation information for the unit. It is used by the enable and disable commands of systemctl(1) during installation.
+
+Once you have generated the systemd unit file, you can copy the generated systemd file to ```/usr/lib/systemd/system``` for installing as a root user and to ```$HOME/.config/systemd/user ``` for installing it as a non-root user. Now, Enable the copied one or more units using `systemctl enable`.
+
+Note: Coping a unit to ```/usr/lib/systemd/system``` and enabling it marks the unit to be automatically started at boot. And smillarly, coping a unit to ```$HOME/.config/systemd/user ``` and enabling it marks the unit to be automatically started on user login.
+
+
+```
+# Generated systemd files.
+$ podman pod create --name systemd-pod
+$ podman create --pod systemd-pod alpine top
+$ podman generate systemd --files --name systemd-pod
+
+# Copy all the generated files.
+
+$ sudo cp pod-systemd-pod.service container-great_payne.service /usr/lib/systemd/system
+$ systemctl enable pod-systemd-po.service
+Created symlink /etc/systemd/system/multi-user.target.wants/pod-systemd-po.service → /usr/lib/systemd/system/pod-systemd-po.service.
+Created symlink /etc/systemd/system/default.target.wants/pod-systemd-po.service → /usr/lib/systemd/system/pod-systemd-po.service.
+$ systemctl is-enabled pod-systemd-po.service
+enabled
+```
+To run the user services placed in `$HOME/.config/systemd/user/` on first login of that user, enable the service with --user flag.
+
+```
+$ systemctl --user enable <.service>
+```
+The systemd user instance is killed after the last session of the user is closed. One can keep the systemd user instance running even after user logs out by enabling `lingering` using
```

Comment:
"of the user" -> "for the user"

Reply:
Second sentence to -> "The systemd user instance can be kept running ever after the user logs out ..."